cfme/fixtures/v2v_fixtures.py

```diff
@@ -212,8 +212,8 @@ def vddk_url():
     return url
 
 
-def get_conversion_data(target_provider):
-    if target_provider.one_of(RHEVMProvider):
+def get_conversion_data(appliance, target_provider):
+    if target_provider.one_of(RHEVMProvider) and appliance.version < '5.11':
```

Comment:
@nachandr : One more change , let's do this instead of checking fro RHEVM twice

      if target_provider.one_of(RHEVMProvider): 
          if appliance.version == '5.11': 
              code ...for 5.11
          else: 
              code for 5.10

Reply:
I agree here, this will be a bit more readable.

Reply:
Making this **Required**